ckanext/geocat/utils/csw_mapping.py

```diff
@@ -53,12 +53,12 @@
                 '//gmd:language/gmd:LanguageCode/@codeListValue']
 GMD_SPATIAL = '//gmd:identificationInfo//gmd:extent//gmd:description/gco:CharacterString/text()'  # noqa
 GMD_PUBLISHER = [
-    '//gmd:identificationInfo//gmd:pointOfContact[.//gmd:CI_RoleCode/@codeListValue = "publisher"]',  # noqa
-    '//gmd:identificationInfo//gmd:pointOfContact[.//gmd:CI_RoleCode/@codeListValue = "owner"]',  # noqa
-    '//gmd:identificationInfo//gmd:pointOfContact[.//gmd:CI_RoleCode/@codeListValue = "pointOfContact"]',  # noqa
-    '//gmd:identificationInfo//gmd:pointOfContact[.//gmd:CI_RoleCode/@codeListValue = "distributor"]',  # noqa
-    '//gmd:identificationInfo//gmd:pointOfContact[.//gmd:CI_RoleCode/@codeListValue = "custodian"]',  # noqa
-    '//gmd:contact//che:CHE_CI_ResponsibleParty',  # noqa
+    '//gmd:identificationInfo//gmd:pointOfContact[.//gmd:CI_RoleCode/@codeListValue = "publisher"]//gmd:organisationName',  # noqa
+    '//gmd:identificationInfo//gmd:pointOfContact[.//gmd:CI_RoleCode/@codeListValue = "owner"]//gmd:organisationName',  # noqa
+    '//gmd:identificationInfo//gmd:pointOfContact[.//gmd:CI_RoleCode/@codeListValue = "pointOfContact"]//gmd:organisationName',  # noqa
+    '//gmd:identificationInfo//gmd:pointOfContact[.//gmd:CI_RoleCode/@codeListValue = "distributor"]//gmd:organisationName',  # noqa
+    '//gmd:identificationInfo//gmd:pointOfContact[.//gmd:CI_RoleCode/@codeListValue = "custodian"]//gmd:organisationName',  # noqa
+    '//gmd:contact//che:CHE_CI_ResponsibleParty//gmd:organisationName',  # noqa
```

Comment:
Shouldn't we point also to the actual name `gco:CharacterString` ? here or in the line 71

Reply:
No I don't think so. I went back to the code that was there before and retrieved the publisher name as before: see here: https://github.com/opendata-swiss/ckanext-geocat/blob/896988ed670aaaa73fb04c9cc56b61b134f88575/ckanext/geocat/utils/csw_mapping.py#L55

Since the task was to just get the url as an additional value I went back to the previous way how we retrieved the name. Does that answer your question?

Reply:
yes, is fine with me, thanks